.github/workflows/test.yml

```diff
@@ -122,7 +134,7 @@ jobs:
         run: >
           source .venv/bin/activate;
           pytest --durations=0 --cov-report=term --cov-report=html:htmlcov --cov-report=xml:coverage.xml --cov=schematic/
-          -m "not (rule_benchmark or table_operations)" --reruns 2 -n auto
+          -m "not (rule_benchmark)" --reruns 2 -n 4
```

Comment:
I swapped this to use `-n 4` instead of `-n auto`

https://pytest-xdist.readthedocs.io/en/latest/distribution.html#running-tests-across-multiple-cpus

I printed out the runner CPU information and it is using a 4 core CPU, but for some reason, using `-n auto` was setting it to use 2 workers. By manually bumping this up we should be getting a bit of a time improvement for tests running.